source/Calamari.Tests/KubernetesFixtures/KubernetesRunsScriptFixture.cs

```diff
@@ -404,6 +413,13 @@ void InstallAwsCli(InstallTools tools)
             Task.Run(() => installAwsCliTask);
             Task.WaitAll(installAwsCliTask);
         }
+        
+        void InstallGCloud(InstallTools tools)
+        {
+            var installGCloud = tools.InstallGCloud();
+            Task.Run(() => installGCloud);
+            Task.WaitAll(installGCloud);
+        }
```

Comment:
I know this is done in other methods as well, but it looks a bit strange to me. Maybe we could test using `async`/`await` instead? I don't think I fully understand the current pattern 🤔 Looks like it is trying to use `Task.Run` and `Task.WaitAll` to avoid using `async`, but I'm not sure what's the reason behind

Reply:
Using Async await here meant the calling signature becomes `InstallTools(async (tools) => await InstallGCloud(tools));` There might be a cleaner way that I'm missing but went with this approach to keep the call signature cleaner.